src/screens/crew6/helmsScreen.cpp

```diff
@@ -126,7 +127,7 @@ void HelmsScreen::onDraw(sf::RenderTarget& window)
         energy_display->setValue(string(int(my_spaceship->energy_level)));
         heading_display->setValue(string(my_spaceship->getHeading(), 1));
         float velocity = sf::length(my_spaceship->getVelocity()) / 1000 * 60;
-        velocity_display->setValue(string(velocity, 1) + DISTANCE_UNIT_1K + "/min");
+        velocity_display->setValue(string(velocity, 1) + DISTANCE_UNIT_1K + tr("unit", "/min"));
```

Comment:
```
string(velocity, 1) + DISTANCE_UNIT_1K + tr("unit", "/min")
```
Would be better as:
```
tr("{value} {unit}/min").format({{"value", string(velocity, 1)}, {"unit": DISTANCE_UNIT_1K}})
```
I think. As that would add flexibility to the translator, and a bit more context where this is used.

Reply:
Done. (I guess the **:** was a typo)